client/src/features/search-bar/search-bar.component.js

```diff
@@ -46,9 +46,13 @@ export const SearchBar = ({
   };
 
   const handleClick = event => {
-    textSearch(inputValue, userLocation.lat, userLocation.lon);
-    updateSearchInput('');
-    toggleAutoComplete(false);
+    if (!inputValue.trim()) {
+      alert("Ops... Please, enter a location name to search the place!")
```

Comment:
I think its better to have the toast component handle all error messages instead of showing alert window for better user experience

Reply:
Good idea! Let's keep if for now since I'm holding the toast feature with reducers and actions. I'm having a problem with combineEpics that Della was looking into yesterday. I'll ask for help fixing it on Slack since we don't have that much time.

Reply:
For sure, alert is not the best solution. I said this in my comment above and yesterday I added a task on Trello for this.